ticdc/deploy-ticdc.md

```diff
@@ -58,3 +58,4 @@ The following are descriptions of options available in the `cdc server` command:
 - `ca`: The path of the CA certificate file used by TiCDC, in the PEM format (optional).
 - `cert`: The path of the certificate file used by TiCDC, in the PEM format (optional).
 - `key`: The path of the certificate key file used by TiCDC, in the PEM format (optional).
+- `config`: The address of the configuration file that TiCDC uses. This option is supported since TiCDC v5.0.0. This option can be used in the TiCDC deployment since TiUP v1.4.0.
```

Comment:
In https://github.com/pingcap/docs-cn/pull/5908/files, line 61 mentions that this item is 可选项。

```suggestion
- `config`: The address of the configuration file that TiCDC uses (optional). This option is supported since TiCDC v5.0.0. This option can be used in the TiCDC deployment since TiUP v1.4.0.
```
.